jenkins_pipelines/environments/manager-qe-continuous-build-validation-NUE

```diff
@@ -34,13 +34,13 @@ node('sumaform-cucumber') {
             // Temporary: should move to uyuni-project
             string(name: 'terracumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/terracumber.git', description: 'Terracumber Git Repository'),
             string(name: 'terracumber_ref', defaultValue: 'master', description: 'Terracumber Git ref (branch, tag...)'),
-            string(name: 'product_version', defaultValue: 'head', description: 'Server product version'),
```

Comment:
I would prefer to keep using head or uyuni master in our continuous validation.

Reply:
It was decided not to use uyuni for BV (time consuming to stabilize).